src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { TODO } from '../../types/todo';
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
+import { USER } from '../../types/User';
+
+type Props = {
+  todo: TODO;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', { 'TodoInfo--completed': todo.completed })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <UserInfo
+        user={usersFromServer.find(u => todo.userId === u.id) as USER}
```

Comment:
The use of `as USER` for type assertion assumes that `usersFromServer.find()` will always find a user. If no user is found, this will result in a runtime error. Consider handling the case where `find` returns `undefined` to ensure the component does not break.